packages/autorest.python/autorest/preprocess/__init__.py

```diff
@@ -282,6 +291,14 @@ def update_parameter(self, yaml_data: Dict[str, Any]) -> None:
                 .lower()
                 for prop, param_name in yaml_data["propertyToParameterName"].items()
             }
+        if yaml_data["location"] == "header" and yaml_data["wireName"].lower() in (
+            "repeatability-request-id",
```

Comment:
I think it would be good to have a constant list at the top, so it's clear what parameters we're hiding